exifread/tags/makernote/reconyx.py

```diff
@@ -0,0 +1,76 @@
+from ...utils import make_string
+
+# maker notes for Reconyx Hyperfire cameras (ref PH)
+# ported from https://fossies.org/linux/Image-ExifTool/lib/Image/ExifTool/Reconyx.pm
+TAGS_HYPERFIRE = {
+    0x00: ('MakerNoteVersion', make_string),  # Sometimes binary
+    0x01: ('FirmwareVersion', ),
+    0x04: ('FirmwareDate', ),
+    0x06: ('TriggerMode', ),
+    0x07: ('Sequence', ),
+    0x09: ('EventNumber', ),
+    0x0b: ('DateTimeOriginal', ),
+    0x12: ('MoonPhase', ),
```

Comment:
It does look like some of your tag definitions are missing sub-definitions, for example for moon phases,

https://metacpan.org/source/EXIFTOOL/Image-ExifTool-12.00/lib/Image/ExifTool/Reconyx.pm#L244

would look something like this (not tested):

```suggestion
    0x12: ('MoonPhase', {
            0: 'New',
            1: 'New Crescent',
            2: 'First Quarter',
            3: 'Waxing Gibbous',
            4: 'Full',
            5: 'Waning Gibbous',
            6: 'Last Quarter',
            7: 'Old Crescent',
        }
    ),
```